<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCadangan</class>
 <widget class="QWidget" name="FormCadangan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>331</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="kodeCadanganLabel">
      <property name="text">
       <string>Kode Cadangan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kodeCadanganLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaKapalLabel">
      <property name="text">
       <string>Nama Kapal</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaKapalLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="bulanLabel">
      <property name="text">
       <string>Bulan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="bulanLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tahunLabel">
      <property name="text">
       <string>Tahun</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tahunLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="nomorIUPLabel">
      <property name="text">
       <string>Nomor IUP</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="nomorIUPLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lautLabel">
      <property name="text">
       <string>Laut</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lautLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="luasLabel">
      <property name="text">
       <string>Luas</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="luasLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="dDHLabel">
      <property name="text">
       <string>DDH</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="dDHLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="iDHLabel">
      <property name="text">
       <string>IDH</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="iDHLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="tDHLabel">
      <property name="text">
       <string>TDH</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="tDHLineEdit"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="pDHLabel">
      <property name="text">
       <string>PDH</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="pDHLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Cadangan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
